azure-pipelines.yml

```diff
@@ -101,6 +101,10 @@ stages:
         sudo dpkg -i ../target/debs/bullseye/libyang*1.0.73*.deb
       displayName: "Install dependency"
 
+    - script: |
```

Comment:
To install coherent packages related to YANG, it would be cleaner if you replace "-script" with the comment "# YANG MODELS" to run after libyang package install. If you want to keep the separate script, please add the same comment.

Reply:
I prefer keeping it as a separate step.. Will add a comment.. Step's displayName attribute anyway indicates "Install sonic yangs"